apps/store/extensions/app/social-reviews/modules/reviews-api.js

```diff
@@ -72,14 +72,19 @@ var api = {};
         var actor = review.actor = {};
         actor.id = ReviewUtils.formatUsername(user);
         actor.objectType = 'person';
+        res.addHeader("Content-Type", "application/json");
 
         try {
             var result = ReviewUtils.createUserReview(review);
-            res.addHeader("Content-Type", "application/json");
-            print(result);
+            if (!result.success) {
+                res = ResponseProcessor.buildErrorResponse(res, 400,
+                    'User has already reviewed on the asset , Multiple reviews are not allowed.');
+            } else {
+                print(result);
+            }
         } catch (e) {
             log.error('[user-reviews-api] Unable to create the review', e);
-            res = ResponseProcessor.buildErrorResponse(res, 500, 'An error has occured while creating the review,please check the server logs for more details');
+            res = ResponseProcessor.buildErrorResponse(res, 500, 'An error has occurred while creating the review,please check the server logs for more details');
         }
     };
 
```

Comment:
Thanks @thushara35 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/536cfb02b4ebbf5e1f2a5366870034af2b7b9616)